test/cluster/test_read_repair.py

```diff
@@ -157,51 +124,23 @@ def get_unique_key(self) -> str:
     partition_tombstone_timestamp = None
     partition_live = False
 
-    class PartitionKey:
-        def __init__(self, value):
-            self.value = value
-            self.raw = serialize_key(self.value)
-            self.token = murmur3(struct.pack(">l", self.value))
-
-        def __lt__(self, o):
-            return self.token < o.token
-
     @classmethod
     def get_select_query(self, ks):
         return f"SELECT * FROM {ks}.tbl"
 
     @classmethod
-    def generate_sstable(cls, total_rows: int, live_rows: set[int], dead_timestamp: int, live_timestamp: int,
-                         deletion_time: datetime.datetime) -> list[dict[str, Any]]:
-        partitions = []
-        formatted_deletion_time = deletion_time.strftime("%Y-%m-%d %H:%M:%S")
-        serialized_value = serialize_int(cls.v)
-        pks = sorted([cls.PartitionKey(pk) for pk in range(total_rows)])
-        for pk in pks:
-            partition: dict[str, Any] = {
-                "key": {"raw": pk.raw, "token": pk.token},
-            }
-            if pk.value in live_rows:
-                partition["clustering_elements"] = [
-                    {
-                        "type": "clustering-row",
-                        "key": {"raw": ""},
-                        "marker": {"timestamp": live_timestamp},
-                        "columns": {"v": {
-                            "is_live": True,
-                            "type": "regular",
-                            "timestamp": live_timestamp,
-                            "value": serialized_value,
-                        }},
-                    },
-                ]
-            else:
-                partition["tombstone"] = {"timestamp": dead_timestamp, "deletion_time": formatted_deletion_time}
+    async def write_data(cls, cql, ks, total_rows: int, live_rows: set[int], all_live_rows: set[int]) -> None:
+        insert_stmt = cql.prepare(f"INSERT INTO {ks}.tbl (pk, v) VALUES (?, ?)")
+        insert_stmt.consistency_level = ConsistencyLevel.LOCAL_ONE
 
-            partitions.append(partition)
+        delete_stmt = cql.prepare(f"DELETE FROM {ks}.tbl WHERE pk = ?")
+        delete_stmt.consistency_level = ConsistencyLevel.LOCAL_ONE
 
-        assert len(partitions) == total_rows
-        return partitions
+        for pk in range(total_rows):
+            if pk in live_rows:
+                await cql.run_async(insert_stmt, (pk, cls.v))
+            elif pk not in all_live_rows:
+                await cql.run_async(delete_stmt, (pk,))
```

Comment:
ditto asyncio.gather

Reply:
The test depends on the relative order of these writes, cannot use `asyncio.gather` here.